From ed14dcf873775dcfc56883f55c10619779d8f87e Mon Sep 17 00:00:00 2001
From: Nicole Mazzuca <mazzucan@outlook.com>
Date: Thu, 24 Sep 2020 09:48:16 -0700
Subject: [PATCH] rename _castu32_f32 to gemm_castu32_f32

since _castu32_f32 is reserved, an implementation is allowed to name a function that,
and it turns out Xcode 12 did.
---
 src/gemm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gemm.c b/src/gemm.c
index 0b06d98..bc2aef3 100644
--- a/src/gemm.c
+++ b/src/gemm.c
@@ -546,6 +546,7 @@ static inline float _mm256_extract_float32(__m256 a, const int index) {
 #include <smmintrin.h>
 #include <cpuid.h>
 
+#define _castu32_f32 gemm_castu32_f32
 static inline float _castu32_f32(uint32_t a) {
     return *((float *)&a);
 }
-- 
2.24.3 (Apple Git-128)

